packages/manager/modules/notifications-sidebar/src/controller.js

```diff
@@ -31,18 +37,46 @@ export default class NotificationsCtrl {
     this.isLoading = true;
     this.numberOfActiveNotifications = 0;
 
+    // Will be bound to the click event on $document
+    this.readAllNotifications = () => {
+      this.toggle = false;
+
+      // Automatically set all unread messages to read
+      // When we close the notifications menu
```

Comment:
**suggestion:** There is no benefit of starting new comment line with a capital letter.

```suggestion
      // when we close the notifications menu
```